src/commands/codepush/deployment/ls.ts

```diff
@@ -0,0 +1,30 @@
+import { AppCommand, CommandArgs, CommandResult, help, failure, ErrorCodes, success, getCurrentApp } from "../../../util/commandline";
+import { out } from "../../../util/interaction";
+import { DefaultApp } from "../../../util/profile";
+import { inspect } from "util";
+import { MobileCenterClient, models, clientRequest } from "../../../util/apis";
+const _ = require("lodash");
+
+const debug = require("debug")("mobile-center-cli:commands:codepush:deployments:ls");
+
+@help("List the deployments associated with an app")
+export default class LsCommand extends AppCommand {
```

Comment:
You are duplicating all this logic between two classes. At minimum we need to remove one command or the other, at most you should extract the logic into a standard base class. I would recommend removing 'ls' and only keep 'list' so it matches the rest of the CLI naming conventions.